README.md

```diff
@@ -1,4 +1,5 @@
 <h1 align="center">
+
```

Comment:
Hello @jurre! 👋🏻 
I'm coming from this repo's README page to let you know the addition of this empty line in the README file broke how the image displays.
You'll find more details in my PR that attempts to fix that: https://github.com/dependabot/dependabot-core/pull/9095.